src/plugins/intel_npu/src/backend/src/zero_device.cpp

```diff
@@ -167,11 +166,9 @@ ov::device::Type ZeroDevice::getDeviceType() const {
     return ov::device::Type::INTEGRATED;
 }
 
-std::shared_ptr<SyncInferRequest> ZeroDevice::createInferRequest(
-    const std::shared_ptr<const ICompiledModel>& compiledModel,
-    const std::shared_ptr<IExecutor>& executor,
-    const Config& config) {
-    return std::make_shared<ZeroInferRequest>(_initStructs, compiledModel, executor, config);
+std::shared_ptr<SyncInferRequest> ZeroDevice::createInferRequest(std::shared_ptr<const ICompiledModel> compiledModel,
+                                                                 const Config& config) {
+    return std::make_shared<ZeroInferRequest>(_initStructs, compiledModel, config);
```

Comment:
Here you need to do "std::move(compiledModel)" as well

Reply:
done, thx